django_emarsys/migrations/0003_newevent.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_emarsys', '0002_neweventinstance'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='NewEvent',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('name', models.CharField(unique=True, max_length=1024)),
+                ('emarsys_id', models.IntegerField()),
+            ],
+            options={
+                'ordering': ['name'],
+                'permissions': [('can_trigger_event', 'Can trigger emarsys events.')],
+            },
+            bases=(models.Model,),
+        ),
+    ]
```

Comment:
No, but it just feels like a bad idea to do it all in one.

Just checked whether there's consensus about this question in general:
https://docs.djangoproject.com/en/1.8/topics/migrations/#data-migrations
```
Migrations that alter data are usually called “data migrations”; they’re best written as separate migrations, sitting alongside your schema migrations.
```

Oh, as a more concrete point here: I didn't migrate `Event`s and had no way to migrate `EventData`, that means reversing the data migration *will* fail, if `Event` and `EventData` are not restored based on fixtures.

I know the reverse migrations are not a huge focus, but I'll make a note of this in the migration file anyway.